Create CONTRIBUTING - RUN UI LOCALLY.md

mkz212 opened the pull request:
@donavanbecker @bwp91 

- Thanks for your help. I think that such a guide will be useful to anyone who would like to run locally.
- Please correct and complete.

mkz212 committed:
Create CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

mkz212 committed:
Update CONTRIBUTING - RUN UI LOCALLY.md

bwp91 commented:
rebased onto alpha for now, i think this will be a good inclusion, just will need some time to review it properly 👍

donavanbecker committed:
Attach Artifacts to Workflow

- allow for a backup just in case the upload to release asset fails

donavanbecker committed:
update issue templates

SonnyT72 committed:
Update fr.json (#1974)

donavanbecker committed:
Merge branch 'beta-4.55.1' into patch-7

donavanbecker committed:
Other Changes

- improve github actions
- removed force of earlier version of `bonjour-service`` as a fix has been implemented in latest version
- updated release drafter template so releases do not start with a `v`

h

mkz212 committed:
improve power options button display (#1961)

bwp91 committed:
update credits modal with supporters

bwp91 committed:
Merge branch 'beta-4.55.2' into pr/1962

bwp91 committed:
Update CONTRIBUTING.md

bwp91 commented:
Hi @mkz212 

sorry for pushing directly to your branch. but you can see that I have updated the existing CONTRIBUTING file with a new section called 'local setup (extended)'

I was thinking maybe you could slot your helpful guide into this section?

what do you think?

mkz212 commented:
> Hi @mkz212
> 
> sorry for pushing directly to your branch. but you can see that I have updated the existing CONTRIBUTING file with a new section called 'local setup (extended)'
> 
> I was thinking maybe you could slot your helpful guide into this section?
> 
> what do you think?

I don't see this section. And I think it would be better as a separate file. Anyway, I'm not a good person for this because I haven't managed to run it until now.
